### ISSUE TITLE
MonetDBLite removed from CRAN

### ISSUE THREAD

**Issue by poldham** [ropensci/taxadb#52]
Hi Carl and collaborators, I was watching the recent rOpenSci community call and was very excited to give taxadb a whirl. 

It seems there is a dependency on MonetDBLite and it is not available for R 3.5.3 (or 3.6 I guess). The CRAN status on the package reveals that it has been removed from CRAN and archived in April 2019 at the request of the maintainer.

I see now that you have spotted this here: [https://github.com/MonetDB/MonetDBLite-R/issues/38](https://github.com/MonetDB/MonetDBLite-R/issues/38) and mentioned the development of duckdb as an alternative. 

Leaving this here for others who will have this question. taxadb looks great from Kari's presentation and I'll look forward to using taxadb when there has been some time to address the dependency issue. 

All the best, 

Paul

**Comment by cboettig** [comment]
Hi @poldham, yup, thanks for the note.  For the moment I've just added `MonetDBLite` under Remotes, so `taxadb` should continue to install successfully with `devtools::install_github()`.  

As you've noted, `MonetDBLite` was removed from CRAN, and the maintainer intends to replace MonetDBLite with `duckdb`, so we'll plan to migrate to that as well.  They are all DBI-based interfaces, so the changes required are pretty minimal. (`taxadb` already supports user-selected databases anyway, `MonetDBLite` is just provided as the default since it is easier to install and faster than alternatives like Postgres).  We'll have to wait until `duckdb` is on CRAN to get `taxadb` to CRAN, but hopefully it is soon.
